ci/build_setup.sh

```diff
@@ -60,7 +60,7 @@ export USER=bazel
 export TEST_TMPDIR=/build/tmp
 export BAZEL="bazel"
 # Not sandboxing, since non-privileged Docker can't do nested namespaces.
-BAZEL_OPTIONS="--package_path %workspace%:${ENVOY_SRCDIR}"
+BAZEL_OPTIONS=""
```

Comment:
I'm pretty sure we still need this since ci/WORKSPACE is used for CI runs.

Reply:
Yeah, we invoke inside the `ci` dir, but need to know where to find the root IIRC.

Reply:
Oh, I see. Done.